.changeset/fluffy-bikes-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/react': patch
```

Comment:
Should this be minor?

Reply:
Maybe? The way I read this ticket is we're fixing a mismatch between the announcement of the outcome from pressing the interactive element. i.e. we announce “Select all rows, checkbox, partially checked, **to check**, press the spacebar.” but when a user presses it, they actually deselect everything.

Reply:
I think that's the accessibility bug which few people knew about, but we're actually changing how this works for everyone

Reply:
I also don't think it really matters either